_posts/2017-06-09-notre-wiki.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Notre Wiki
+authors:
+  - laurent.bossavit
+---
+Dans certaines entreprises on distribue un « manuel de l'employé ». Il permet de s'orienter, de savoir quoi faire le premier jour et où poser ses affaires. Nous pensons qu'un « manuel de l'employé » est une excellente idée, et qu'elle est meilleure encore s'il est collaboratif et ouvert.
+
+À l'image d'un certain nombre d'entreprises du privé, comme [Basecamp](https://github.com/basecamp/handbook) ou [Valve](http://www.valvesoftware.com/company/Valve_Handbook_LowRes.pdf), nous avons dès le début décidé de le [rendre public](https://github.com/sgmap/beta.gouv.fr/wiki/). Si vous envisagez de travailler chez nous, ou de travailler avec nous, il vous permettra de savoir à quoi vous attendre, avant même le premier jour.
```

Comment:
Et en plus, l'important et de savoir pourquoi on l'a fait. Je cite :

> « For over 10 years, we didn’t have a handbook. In those 10 years, when a new person joined the company, they were expected to figure things out for themselves. But when we grew from a company of 10, 20, 30 employees to a company of over 50, our “introduction by immersion” style stopped working. New hires felt lost and isolated, and their first weeks or even months on the job were stressful because of it. It can be unnerving to join any company, but perhaps Basecamp in particular, given how different some of our practices are. There’s as much to unlearn as there is to learn. Prior to this handbook, navigating that path was both somewhat random and almost entirely oral. For a company that prides itself on clarity and writing, that just wasn’t right. »

Il me semble que nous n'avons pas eu besoin qu'on nous apprend ça, n'est-ce pas ?